packages/apidom-ls/src/config/openapi/openapi3_1/meta.ts

```diff
@@ -1,8 +1,12 @@
+import completion from './completion';
 import documentation from './documentation';
+import lint from './lint';
 import { FormatMeta } from '../../../apidom-language-types';
 
 const meta: FormatMeta = {
+  completion,
   documentation,
+  lint,
```

Comment:
This is un-important nitpick, that I mention only because of code consistency. The order by convention that we use is:

```js
lint,
completion,
documentation,
```

Reply:
yea, I went with alphabetical order. will change.